Git-commit: 5c472e201b8ee4f9da3908390a2d837cc9849367
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 2 Mar 2020 14:40:19 +0000
Subject: [PATCH] drivers: char: vcio: Use common compat header
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

The definition of compat_ptr is now common for most platforms, but
requires the inclusion of <linux/compat.h>.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 drivers/char/broadcom/vcio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/char/broadcom/vcio.c b/drivers/char/broadcom/vcio.c
index d2598663a2b5..a39155a94fb7 100644
--- a/drivers/char/broadcom/vcio.c
+++ b/drivers/char/broadcom/vcio.c
@@ -18,6 +18,7 @@
 #include <linux/module.h>
 #include <linux/slab.h>
 #include <linux/uaccess.h>
+#include <linux/compat.h>
 #include <soc/bcm2835/raspberrypi-firmware.h>
 
 #define MBOX_CHAN_PROPERTY 8
-- 
2.31.1

